lldb/source/Target/TargetProperties.td

```diff
@@ -10,6 +10,9 @@ let Definition = "target_experimental" in {
   def SwiftUseReflectionSymbols : Property<"swift-use-reflection-symbols", "Boolean">,
     Global, DefaultTrue,
     Desc<"if true, optimize the loading of Swift reflection metadata by making use of available symbols.">;
+  def SwiftReadMetadataFromDSYM: Property<"swift-read-metadata-from-dsym", "Boolean">,
```

Comment:
It's great that you made this switchable, but dsymutil only copies reflection metadata if the linker has stripped it, so there's little risk in pessimizing existing workflows with this feature.